src/pani/registry.cljs

```diff
@@ -0,0 +1,18 @@
+(ns pani.registry)
+
+(defonce unsubs (atom {}))
+
+(defn register-listener [ref type unsub!]
+  (swap! unsubs update-in [(str ref) type] #(set (conj % unsub!))))
+
+(defn- flatten-vals [xs]
+  (if-not (map? xs)
+    xs
+    (reduce into (map flatten (vals xs)))))
+
+(defn- call-all! [fs] (doseq [f fs] (f)))
+
+(defn disable-unsubs! [& path]
+  (swap! @unsubs update-in
+         (map str path)
+         (comp call-all! flatten-vals)))
```

Comment:
Damn, you're right :japanese_goblin:. Also the `@` should be in that swap :blush: